Apps/Admin/Tests/Functional/cypress/integration/e2e/analytics.cy.js

```diff
@@ -18,10 +20,9 @@ describe("System Analytics", () => {
 
         cy.intercept("GET", "**/CsvExport/GetRatings").as("getRatings");
 
-        cy.intercept(
-            "GET",
-            "**/CsvExport/GetInactiveUsers?timeOffset=420&inactiveDays=100"
```

Comment:
Reason why offset and inactiveDays were specified was to ensure that those values were being read in the intercept vs just *.